src/mercury_engine_data_structures/game_check.py

```diff
@@ -102,6 +102,18 @@ def is_sr_or_else(subcon1, subcon2) -> IfThenElse:
     return IfThenElse(construct.this._params.target_game == Game.SAMUS_RETURNS.value, subcon1, subcon2)
 
 
+def is_at_most(target: Game, subcon1, subcon2) -> IfThenElse:
```

Comment:
I prefer the terms `subcontrue` and `subconfalse` just to make it clearer when each subcon is getting used.